gatsby/src/components/row/row.tsx

```diff
@@ -27,6 +27,7 @@ const Row: React.FC<IProps> = ({
     return strng;
   };
 
+  // row
   return <div className={`row ${getClassNames()}`}>{children}</div>;
```

Comment:
přemýšlím, jestli je tohle nezbytné – těžko říct, protože tam máme dva divy s className="row"

Reply:
Pravda, zkusím to dát pryč

Reply:
Funguje